test/PolyhedralGeometry/cone.jl

```diff
@@ -136,30 +136,33 @@
 
   @testset "constructors" begin
     @test cone_from_inequalities(f, [-1 0 0; 0 0 -1]) == Cone2
-    @test cone_from_inequalities(f, [-1 0 0; 0 0 -1]; non_redundant = true) == Cone2
+    @test cone_from_inequalities(f, [-1 0 0; 0 0 -1]; non_redundant=true) == Cone2
     @test cone_from_inequalities(f, facets(Cone4), linear_span(Cone4)) == Cone4
-    @test cone_from_inequalities(f, facets(Cone4), linear_span(Cone4); non_redundant = true) == Cone4
-    @test cone_from_equations(f, [0 1 0]) == cone_from_inequalities(f, Matrix{Int}(undef, 0, 3), linear_span(Cone4))
+    @test cone_from_inequalities(
+      f, facets(Cone4), linear_span(Cone4); non_redundant=true
+    ) == Cone4
+    @test cone_from_equations(f, [0 1 0]) ==
+      cone_from_inequalities(f, Matrix{Int}(undef, 0, 3), linear_span(Cone4))
   end
 end
 
 @testset "transform $T" for (f, T) in _prepare_scalar_types()
   pts = [1 0 0 0; 1 1 0 0; 1 1 1 0; 1 0 1 0]
   lin = [0 0 0 1]
-  A = [ -6    3   1    3;
-       -24    7   4   13;
-       -31    9   5   17;
-       -37   11   6   20]
+  A = [-6 3 1 3;
+    -24 7 4 13;
+    -31 9 5 17;
+    -37 11 6 20]
   # We also put inverse here, since inversion in Julia produces float errors
   # and we don't want to do complicated matrix conversions here.
-  invA = [1   2   3   -4;
-          1   0   1   -1;
-          1   9   0   -6;
-          1   1   5   -5]
+  invA = [1 2 3 -4;
+    1 0 1 -1;
+    1 9 0 -6;
+    1 1 5 -5]
```

Comment:
I am not a fan of how this new option formats matrices like this. The offset of the first row is weird. I think without this option this was more aligned.
But let's hear other peoples opinions as well.